src/components/button/button-small/button-small.styles.tsx

```diff
@@ -0,0 +1,19 @@
+import { createUseStylesConfig } from '../../../styles/create-use-styles';
+import { formatSize } from '../../../styles/format-size';
+import { ButtonStyleConfig } from '../button-style.config';
+
+export const useButtonSmallStyles = createUseStylesConfig<ButtonStyleConfig>(({ colors, typography }) => ({
```

Comment:
set `ButtonStyleConfig` as `createUseStylesConfig` default generic value (it's used in most cases)